satellite/console/service.go

```diff
@@ -241,6 +257,101 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 	return pp, nil
 }
 
+// AddNewUserPaymentMethod creates new payment method at stripe
+func (s *Service) AddNewUserPaymentMethod(ctx context.Context, paymentMethodToken string) (paymentMethod *UserPaymentMethod, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	authorization, err := GetAuth(ctx)
```

Comment:
just an observation - in other service methods that requires authentication we don't place empty line between 
defer mon.Task()(&ctx)(&err) and authorization, err := GetAuth(ctx)